cypress/support/search_page.js

```diff
@@ -0,0 +1,69 @@
+export const Selectors = {
+    sortSelectBox: '[data-testid=sort-select]',
+    datasetsFoundCount: '[data-testid=result-count]',
+    paginator: '[data-testid=paginator]',
+    dialogContent: '[data-testid=dialog-content]',
+    search: '[data-testid=search]',
+    datasets: '[data-testid=dataset-list]',
+    firstDataset: '[data-testid=dataset-list] > :nth-child(1)',
+    apiAccessibleCheckbox: '[data-testid="checkbox-indicator-API Accessible"]',
+    organizations: '[data-testid=facet-list-organization]',
+    cogoCheckBox: '[data-testid="checkbox-indicator-COGO (5)"]',
+    keywords: '[data-testid=facet-list-keywords]',
+    bicycleCheckBox: '[data-testid="checkbox-indicator-bicycle (5)"]'
+}
+
+export const Routes = {
+    routes: {
```

Comment:
Is it necessary to nest these in `routes`? It seems like it would read easier to to `Routes.allDatasetsLastModified` compared to `Routes.routes.allDatasetsLastModified`

Reply:
ha, good point I'll change it!